src/Auditor.jl

```diff
@@ -209,6 +209,33 @@ function audit(prefix::Prefix, src_name::AbstractString = "";
         rm(f; force=true)
     end
 
+    # Make sure that `prefix` in pkg-config files *.pc points to the right
+    # directory.
+    pc_files = collect_files(prefix, endswith(".pc"))
+    for fs in pc_files
+        # Make sure the file still exists on disk
+        if !isfile(fs)
+            continue
+        end
+
+        fs = open(file, "r")
```

Comment:
You're overriding `fs`?  Did you mean to swap `fs` and `file`?  Also, for what is worth I find the
```julia
open(...) do ...
   ...
end
```
syntax nicer as you don't need to manually close the file (and it's guaranteed to be closed in case of failure)

Reply:
Thanks! Should be fixed.